config/jvm.options

```diff
@@ -33,7 +33,7 @@
 ################################################################
 
 ## GC configuration
--XX:+UseConcMarkSweepGC
+-XX:+UseG1GC
```

Comment:
why are we changing this ? 

Looking at the shipped default jvm.options i can see the old settings

```
## GC configuration     
-XX:+UseConcMarkSweepGC         
-XX:CMSInitiatingOccupancyFraction=75
-XX:+UseCMSInitiatingOccupancyOnly 

```

Reply:
UseConcMarkSweepGC is deprecated in the version of Java we're using and will be removed from a future version. The recommendation is to migrate to the G1 collector - http://openjdk.java.net/jeps/291.

Reply:
sure but the upstream image is still using the same settings , see the snippet taken from the upstream official image that we base our image on, so i would rather keep referencing the upstream image

Reply:
we will change it on whatever update theupstream image changes it , can you please compare the "jvm.options" from upstream with ours ? maybe we are in a position to drop our override ?

Reply:
Only our settings for the DNS settings are different. I've reverted everything else back to the upstream defaults.